LarkatorGUI/MainWindow.xaml.cs

```diff
@@ -476,6 +477,7 @@ private void CreateSearch_Click(object sender, RoutedEventArgs e)
 
             speciesCombo.ItemsSource = arkReader.AllSpecies;
             groupsCombo.ItemsSource = ListSearches.Select(sc => sc.Group).Distinct().OrderBy(g => g).ToArray();
+            groupsCombo.SelectedIndex = Properties.Settings.Default.lastGroup;
```

Comment:
Nice idea.

Reply:
Currently, I store the index from the dropbox, but it might be better to store the actual group name itself?  Initially, I thought that would cause problems, but since the group would be added if it didn't exist, using the actual group name should be more consistent.

Reply:
Switched to string variable in commit c2de84342ccc47e9a37306f381356cb72d0e3f12